presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -593,7 +594,12 @@ public HiveSessionProperties(HiveClientConfig hiveClientConfig, OrcFileWriterCon
                         MATERIALIZED_VIEW_MISSING_PARTITIONS_THRESHOLD,
                         "Materialized views with missing partitions more than this threshold falls back to the base tables at read time",
                         hiveClientConfig.getMaterializedViewMissingPartitionsThreshold(),
-                        true));
+                        true),
+                stringProperty(
+                        METASTORE_HEADERS,
+                        "Tetris is something specific to facebook . How about this : The headers that will be sent in the calls to Metastore",
```

Comment:
lol
`The headers that will be sent in the calls to Metastore`

Reply:
🤣